com.github.juzzlin.DustRacing2D.yml

```diff
@@ -0,0 +1,36 @@
+app-id: com.github.juzzlin.DustRacing2D
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: dustrac-game
+rename-icon: dustrac-game
+rename-desktop-file: dustrac-game.desktop
+rename-appdata-file: dustrac.appdata.xml
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-config/.dustrac
```

Comment:
```suggestion
```

This is not needed either, the app respects XDG_CONFIG_HOME.

Reply:
Removed it.